docs/conf/helpop-full.conf.example

```diff
@@ -1040,42 +1056,42 @@ setting +I <extban>.
 Matching extbans:
 
  j:<channel>   Matches anyone in the given channel. Does not support
-               wildcards (requires channelban module).
- r:<realname>  Matches users with a matching realname (requires gecosban
-               module).
- s:<server>    Matches users on a matching server (requires serverban
-               module).
+               wildcards (requires the channelban module).
+ r:<realname>  Matches users with a matching realname (requires the
+               gecosban module).
+ s:<server>    Matches users on a matching server (requires the
+               serverban module).
  z:<certfp>    Matches users having the given SSL certificate
-               fingerprint (requires sslmodes module).
+               fingerprint (requires the sslmodes module).
  O:<opertype>  Matches IRCops of a matching type, mostly useful as an
-               an invite exception (requires operchans module).
+               an invite exception (requires the operchans module).
  R:<account>   Matches users logged into a matching account (requires
-               services account module).
+               the services account module).
  U:<banmask>   Matches unregistered users matching the given banmask.
-               (requires services account module).
+               (requires the services account module).
 
 Acting extbans:
 
  c:<banmask>   Blocks any messages that contain color codes from
-               matching users (requires blockcolor module).
- m:<banmask>   Blocks messages from matching users (requires muteban
+               matching users (requires the blockcolor module).
+ m:<banmask>   Blocks messages from matching users (requires the muteban
                module). Users with +v or above are not affected.
  p:<banmask>   Blocks part messages from matching users (requires
-               nopartmsg module).
+               the nopartmsg module).
  A:<banmask>   Blocks invites by matching users even when +A is set
-               (requires allowinvite module).
+               (requires the allowinvite module).
  B:<banmask>   Blocks all capital or nearly all capital messages from
-               matching users (requires blockcaps module).
- C:<banmask>   Blocks CTCPs from matching users (requires noctcp
+               matching users (requires the blockcaps module).
+ C:<banmask>   Blocks CTCPs from matching users (requires the noctcp
                module).
  N:<banmask>   Blocks nick changes from matching users (requires
-               nonicks module).
- Q:<banmask>   Blocks kicks by matching users (requires nokicks
+               the nonicks module).
+ Q:<banmask>   Blocks kicks by matching users (requires the nokicks
                module).
  S:<banmask>   Strips color/bold/underline from messages from matching
```

Comment:
This should probably say.

>Strips formatting codes from messages ...

Reply:
Sure, I'll check/update the same elsewhere for the mode too.